src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -266,7 +266,7 @@ public void setRemoteFs(String remoteFs) {
 
     @Deprecated
     public String getRemoteFs() {
-        return getFirstContainer().map(ContainerTemplate::getWorkingDir).orElse(null);
+        return getFirstContainer().map(ContainerTemplate::getWorkingDir).orElse(ContainerTemplate.DEFAULT_WORKING_DIR);
```

Comment:
If left null, not sure exactly why, but the workspaces still default to `/home/jenkins/workspace` and this breaks all the `container` calls.

Reply:
This method is deprecated; is it still used in production?

Reply:
Yes, unfortunately used by KubernetesSlave constructor. I have some needed refactoring in mind to make this better.